packages/node/src/lib/express-middleware.ts

```diff
@@ -43,6 +44,50 @@ function patchResponse(res) {
   };
 }
 
+/**
```

Comment:
Based on our Slack thread, do we still wanna do all of this stuff? Or is it overkill for now?

Reply:
Chatted offline and we're good to keep this.